CHANGELOG-hmp-122-setup-context-flaskdata.md

```diff
@@ -0,0 +1,2 @@
+- Context was set up in the App file to begin transition of prop drilling to context.
+- Prop drilling to context was completed for Donor page.
```

Comment:
```suggestion
- Pass flask data to Donor page components by context.
```
Small, but I'd like to match the tense of other changelog entries. It's small and not a huge issue if we're at times inconsistent, but worth noting.